docs/install/_initial_setup.rst

```diff
@@ -21,55 +21,58 @@ To do the initial setup of your Kolibri facility after the installation, follow
+		   :alt: In this step you need to choose a type of facility and the name.
 
+		   Choose the type of facility and set the name.
 
-#. **Choose a Facility setup.** Click the link *More information about these settings* for more details about user permissions for each setup type.
 
-+--------------------------------------------------+----------------------------------------------------+
-| Facility type                                    | Users                                              |
-+==================================================+====================================================+
-| | **Self-managed**                               | * Guests can create their own accounts.            |
-| | For parent-child learning, homeschooling       | * Users can edit their account information.        |
-| | or supplementary individual learning.          |                                                    |
-+--------------------------------------------------+----------------------------------------------------+
-| | **Admin managed**                              | * Admins must create all user accounts.            |
-| | For schools and other formal learning contexts.| * Users can sign in without password.              |
-| |                                                | * Users cannot edit their account information.     |
-+--------------------------------------------------+----------------------------------------------------+
-| | **Informal and personal use**                  | * Guests can create their own accounts.            |
-| | For libraries, orphanages, correctional        | * Users can edit their account information.        |
-| | facilities, youth centers, computer labs,      |                                                    |
-| | and other non-formal learning contexts.        |                                                    |
-+--------------------------------------------------+----------------------------------------------------+
+#. **Guest access.** 
 
-  	.. figure:: img/facility-setup.png
-	   :alt: Choose a Facility setup.
+  	.. figure:: img/guest-access.png
+	   :alt: Select if guests can access Kolibri content without the need to create an account.
 
-	   Choose a Facility setup.
+	   Select if guests can access Kolibri content without the need to create an account.
 
-  	.. figure:: img/facility-setup-details.png
-	   :alt: View the Facility setup details.
 
-	   View the Facility setup details.
+#. **User account creation.** 
 
+  	.. figure:: img/sign-up.png
+	   :alt: Select if anyone can create a user account for themself, or if user accounts must be created by Kolibri admins.
 
-4. **Create your Admin account.** This Admin user will be a **Superuser** able to manage all the device content, and all the rest of the facility users and their permissions.
+	   Select if anyone can create a user account for themself, or if user accounts must be created by Kolibri admins.
```

Comment:
"themselves" seems to be more commonly used than "themself"

refs:

* http://canada.justice.gc.ca/eng/rp-pr/csj-sjc/legis-redact/legistics/p1p30.html
* https://blog.oxforddictionaries.com/2013/01/15/themself/